src/managers/EmbraceLogManager/EmbraceLogManager.ts

```diff
@@ -43,9 +47,11 @@ export class EmbraceLogManager implements LogManager {
 
   public logException(
     error: Error,
-    handled: boolean,
-    attributes: Record<string, AttributeValue | undefined> = {},
-    timestamp: number = this._perf.getNowMillis()
+    {
+      handled = false,
```

Comment:
I wonder if handled should be one of the required ones, to make people think what they are actually calling...
But, having said that, I was looking at android https://embrace.io/docs/android/integration/log-message-api/?android-language=kotlin#log-handled-exception and looks like it's not even able to tell that? 🤔